conf/defaults.ini

```diff
@@ -615,6 +615,9 @@ id_response_header_prefix = X-Grafana
 # The header value will encode the namespace ("user:<id>", "api-key:<id>", "service-account:<id>")
 id_response_header_namespaces = user api-key service-account
 
+# Enables the use of managed service accounts for plugin authentication
+managed_service_accounts_enabled = false
```

Comment:
Sample.ini and configuration docs is needed as well I guess?

Reply:
I'll tackle docs and the plugin example in separate PRs and involve the docs squad. :+1: